.config/lints.toml

```diff
@@ -1,9 +1,57 @@
 allow = [
-    # This encourages importing `as` which breaks IDEs
+    # Does not apply to us
+    "clippy::blanket_clippy_restriction_lints",
+    "clippy::exhaustive_enums",
+    "clippy::exhaustive_structs",
+    "clippy::implicit_return",
+    "clippy::missing_docs_in_private_items",
+    "clippy::missing_inline_in_public_items",
+    "clippy::missing_trait_methods", # Should be enabled per `impl` instead of globally
     "clippy::module_name_repetitions",
+    "clippy::pub_use",
+    "clippy::pub_with_shorthand", # # opposite of `clippy::pub_without_shorthand`
+    "clippy::question_mark_used",
+    "clippy::ref_patterns",
+    "clippy::self_named_module_files", # opposite of `clippy::mod_module_files`
+    "clippy::semicolon_outside_block", # opposite of `clippy::semicolon_inside_block`
+    "clippy::separated_literal_suffix", # opposite of `clippy::unseparated_literal_suffix`
+    "clippy::shadow_reuse",
+    "clippy::shadow_same",
+    "clippy::unneeded_field_pattern",
 
-    # The lint is great, but sadly it has too many false positives
-    "clippy::significant_drop_tightening",
+    # Consider enabling:
+    "clippy::absolute_paths",
+    "clippy::default_numeric_fallback",
```

Comment:
in favor of enabling

Reply:
This can get pretty verbose if you do more with numbers but generally, I tend to agree. But this will also require a `_i32` on a simple `+= 1;` assignment which is quite noisy.

Reply:
I thought this would only ever warn when one uses a literal in a case where the default would've been used, e.g. where the type is unclear, but if it's for every literal I agree, it'll be too noisy.